client/blocks/signup-form/index.jsx

```diff
@@ -319,6 +321,14 @@ class SignupForm extends Component {
 
 	handleBlur = () => {
 		const data = this.getUserData();
+		const fieldId = event.target.id;
```

Comment:
The `event` variable here is not defined. The `handleBlur` function should declare `event` as parameter.

The reason why this works and why ESLint doesn't complain is that there is a [`window.event` global](https://developer.mozilla.org/en-US/docs/Web/API/Window/event) that contains the currently handled event. But that's a weird legacy feature of the web platform that is better avoided. Another slight difference is that in one case `event` is the native DOM event, and in the other one (as `handleBlur` param) it's React event wrapper.

Reply:
> We don't want to implement a focusEmail behavior, do we?

That's correct. Showing the email validation error is intentional in this scenario since technically we've already progressed past that field.

> The event variable here is not defined. 

Doh! Another great catch. Updated in https://github.com/Automattic/wp-calypso/commit/9c3896bf4c1db37013000bd879b90e6570b551fd Thank you!

> The reason why this works and why ESLint doesn't complain is that there is a window.event global that contains the currently handled event. But that's a weird legacy feature of the web platform that is better avoided.

I did not know this. Thanks for the heads up!

Reply:
> I did not know this. Thanks for the heads up!

Neither did I until earlier today 🙂